src/core/agent/agent.ts

```diff
@@ -296,12 +296,32 @@ class Agent {
     this.agentServicesProps.signifyClient = this.signifyClient;
     await this.connectSignifyClient();
 
+    await this.basicStorage.save({
+      id: MiscRecordId.PROCESS_RECOVERING,
+      content: { value: true },
```

Comment:
To make this extensible in the future, perhaps we can do `{ value: { syncing: true } }`